src/main/resources/PatientRegistration.fxml

```diff
@@ -9,7 +9,7 @@
 <?import javafx.scene.control.*?>
 <?import javafx.scene.layout.*?>
 
-<BorderPane maxHeight="-Infinity" maxWidth="-Infinity" minHeight="-Infinity" minWidth="-Infinity" prefHeight="745.0" prefWidth="1136.0" xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1" fx:controller="medicaldoctor.PatientRegistrationController">
+<BorderPane maxHeight="-Infinity" maxWidth="-Infinity" minHeight="-Infinity" minWidth="-Infinity" prefHeight="745.0" prefWidth="1136.0" xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1" fx:controller="medicaldoctor.controllers.navbar.patientRegistrationController">
```

Comment:
i don't understand this `navbar package`.... there is only one navbar controller

Reply:
anything that is in the navbar goes there. It's to clean things up for me

Reply:
anything that is in the navbar is not a class thus is not in a package.
patient registration is not in the navbar either????